electronics/Arduino/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            Arduino
+version         1.8.8
+revision        0
+categories      electronics
+platforms       darwin
+maintainers     {gmail.com:giansalvo.gusinu} openmaintainer
+license         GPL-2
+
+description     Arduino Software (IDE) makes it easy to write code and upload it to the board. 
+
+long_description \
+    Arduino is an open-source electronics platform based on easy-to-use \
+    hardware and software. It's intended for anyone making interactive \
+    projects. The Arduino IDE runs on Windows, Mac OS X, and Linux. \
+    The environment is written in Java and based on Processing and other \
+    open-source software. This software can be used with any Arduino board.
+
+homepage        https://www.arduino.cc/
+
+master_sites    https://github.com/arduino/Arduino/archive/
+distfiles       ${version}.tar.gz
```

Comment:
This is usually accomplished by
```
# put this immediately after PortSystem
PortGroup github 1.0

# instead of setting name, version, master_sites, distfiles, just do this
github.setup arduino Arduino 1.8.8
```

Reply:
At first I tryed that way but it didn't work.

Reply:
I think you need to add:

`github.tarball_from releases`

Reply:
Using the PortGroup options, neither of these works:
```
github.tarball_from releases
github.tarball_from downloads
```
The file is located in ```archive``` subfolder.

Reply:
I see this:

`https://github.com/arduino/Arduino/releases/download/1.8.8/arduino-1.8.8.tar.xz
`

Reply:
I should test it first, but in this case the following should work:
```
github.setup arduino Arduino 1.8.8 arduino-
githb.tarball_from releases
use_xz yes
```
In the first command one adds the "arduino-" prefix to the tarball name. The second one switches from tags to releases, and the last one switches to `.tar.xz` from the default `.tar.gz`.

Reply:
ok